src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -256,8 +256,8 @@ private void removeRcaRequestHandler() {
             }
           }
         },
-        2 * pollerPeriodicity,
-        pollerPeriodicity,
+        2 * rcaNannyPollerPeriodicity,
```

Comment:
shouldn't it be `rcaConfPollerPeriodicity` ? As the nanny waits for the first read of the conf ?

Reply:
Yeah you are right, I will fix that